matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/algorithms/megolm/MXMegolmEncryption.kt

```diff
@@ -137,11 +137,10 @@ internal class MXMegolmEncryption(
         return MXOutboundSessionInfo(sessionId, SharedWithHelper(roomId, sessionId, cryptoStore))
     }
 
-    /**
-     * Ensure the outbound session
-     *
-     * @param devicesInRoom the devices list
-     */
```

Comment:
This doc is a bit useless but should not be deleted

Reply:
Done.